git-bug, git-cola, gitlint: add page (#6444)
tldr-pages/tldr: pages/common/git-bug.md

# git bug

> > Is a distributed bug tracker embedded in git. It uses git's internal storage so no files are added in your project.

Reviewer: ```suggestion
> A distributed bug tracker that uses git's internal storage, so no files are added in your project.
```

> > As you would do with commits and branches, you can push your bugs to the same git remote you are already using to collaborate with other people.

Reviewer: Thank you for contribution, all examples look good to me. I think this sentence could be shortened but I cannot come up with a precise and shorter one.

Author: Hi, would you like it to be

> You may submit your issues to the same git remote you use to work with others, just like you would commits and branches.

Reviewer: It seems to be more brief. Just go for it if you think this is precise enough.

Reviewer: Remember to add a newline in the end of pages.

Author: Yeah sure, thanks for the suggestions.

> More information: <https://github.com/MichaelMure/git-bug/blob/master/doc/md/git-bug.md>.

- Create a new identity:

`git bug user create`

- Create a new bug:

`git bug add`

- You can push your new entry to a remote:

`git bug push {{remote}}`

- You can pull for updates:

> `git bug pull {{remote}}`

Author: I thought it would be better if we let it be as per the official documentation. Should i remove it?

> `git bug pull {{remote}}`

Author: Sure, will do the necessary changes and push again.

- List existing bugs:

`git bug ls`

- Filter and sort bugs using a query:

`git bug ls "{{status}}:{{open}} {{sort}}:{{edit}}"`

- Search for bugs by text content:

> `git bug ls "{{searchquery}}" baz`

Reviewer: ```suggestion
`git bug ls "{{search_query}}" baz`
```
